OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -80,6 +80,9 @@ public class CargoInfo : TraitInfo, Requires<IOccupySpaceInfo>
 			"A dictionary of [actor id]: [condition].")]
 		public readonly Dictionary<string, string> PassengerConditions = new Dictionary<string, string>();
 
+		[Desc("Change the passengers owner if transport owner changed")]
```

Comment:
I'd suggest a slight tweak in wording: `Change the passengers' owner if the transport's owner changed.`

Reply:
fixed